Module exported incorrectly from dnt

## Bug Report

When you try to build a Next.js project using the `useInk` library the building process fails with the error:

> Module not found: Package path . is not exported from package useink ...

### Expected Behavior

The "[Deno to npm package build tool](https://github.com/denoland/dnt)" should include the right CommonJS module

### Steps To Reproduce
1. Clone https://github.com/paritytech/useink-kitchen-sink
2. `cd frontend`
3. `yarn`
4. `yarn build`

### Environment

- Deno: 1.32.1
- Node: v18.13.0
- Browser: Google Chrome 111
- OS: macOS 13.2.1

--- a/_tasks/dnt.ts
+++ b/_tasks/dnt.ts
@@ -33,7 +33,6 @@ await Promise.all([
       sourceMap: true,
       target: 'ES2021',
     },
-    scriptModule: false,
     shims: {
       deno: {
         test: true,

